From 429e0de59341c04101e07dab5571c0c0070cbd77 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Mon, 8 Jun 2020 13:33:47 +0800
Subject: [PATCH 1/9] arch: arm64: dts: add bus-type property in mipi_csi_x dts
 node

refer to nxp-imx-sdk5.4, commit a22e26b75ab0("arm64: dts:
imx8qm: Change OV10635 as default camera"), add bus-type
property in mipi_csi_x dts node. Otherwise, code will enter wrong
case in kernel v5.2.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
index 05e4d28cb42f..89d762268a4a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
@@ -131,6 +131,7 @@
 		mipi_csi0_ep: endpoint {
 			remote-endpoint = <&ov5640_mipi_ep>;
 			data-lanes = <1 2>;
+			bus-type = <4>;
 		};
 	};
 };
@@ -144,6 +145,7 @@
 		mipi_csi1_ep: endpoint {
 			remote-endpoint = <&ov5640_mipi_ep_1>;
 			data-lanes = <1 2>;
+			bus-type = <4>;
 		};
 	};
 };
-- 
2.17.1

